### DISCUSSION TITLE
Welcome to open-attestation Discussions!

### DISCUSSION THREAD

**Discussion by Nebulis** [Open-Attestation/open-attestation#138] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adjebbar** [comment]
Hi all,
I have used the opencert (open-attestation) solution for some time locally and hosted on a server and I can more or less handle it. I would like to start an activity with this app for a school project. My problem is that I am not experienced in advertising and third party services. Does anyone have good experience of launching a project with opencert blockchain? and if so would you direct me on how to captivate buyers?
